company-lsp.el

```diff
@@ -477,9 +481,5 @@ See the documentation of `company-backends' for COMMAND and ARG."
   (when company-lsp-enable-snippet
     '(:textDocument (:completion (:completionItem (:snippetSupport t))))))
 
-(add-hook 'lsp-before-initialize-hook
```

Comment:
This change is not related to this PR. Please revert it and send a separate one.

Reply:
Done.